clyent/logs/handlers.py

```diff
@@ -54,6 +58,21 @@ def emit(self, record):
             print(msg, file=stream)
 
 
+class JsonStreamHandler(logging.Handler):
```

Comment:
Is there any reason not to use a `formatter` class here?

https://docs.python.org/2/library/logging.html#formatter-objects